iroh-net/src/magicsock.rs

```diff
@@ -1,5 +1,18 @@
 //! Implements a socket that can change its communication path while in use, actively searching for the best way to communicate.
 //!
+//! ### `DEV_DERP_ONLY` env var:
```

Comment:
This is a compile time switch via an env var. I think this should note that the DEV_DERP_ONLY env var needs to be preset at compile time.

https://doc.rust-lang.org/std/macro.option_env.html